docs/quantize.md

```diff
@@ -0,0 +1,95 @@
+# How to Quantize a Model
+
+Sometimes the model you want to work with is not available at [https://ollama.ai/library](https://ollama.ai/library). If you want to try out that model before we have a chance to quantize it, you can use this process.
+
+## Figure out if we can run the model?
+
+Not all models will work with Ollama. There are a number of factors that go into whether we are able to work with the next cool model. First it has to work with llama.cpp. Then we have to have implemented the features of llama.cpp that it requires. And then, sometimes, even with both of those, the model might not work…
+
+1. What is the model you want to convert and upload?
+2. Visit the model’s page on HuggingFace. 
+3. Switch to the **Files and versions** tab.
+4. Click on the **config.json** file. If there is no config.json file, it may not work.
+5. Take note of the **architecture** list in the json file.
+6. Does any entry in the list match one of the following architectures?
+    1. LlamaForCausalLM
+    2. MistralForCausalLM
+    3. RWForCausalLM
+    4. FalconForCausalLM
+    5. GPTNeoXForCausalLM
+    6. GPTBigCodeForCausalLM
+7. If the answer is yes, then there is a good chance the model will run after being converted and quantized.
+8. An alternative to this process is to visit [https://caniquant.tvl.st](https://caniquant.tvl.st) and enter the org/modelname in the box and submit.
+
+At this point there are two processes you can use. You can either use a Docker container to convert and quantize, OR you can manually run the scripts. The Docker container is the easiest way to do it, but it requires you to have Docker installed on your machine. If you don't have Docker installed, you can follow the manual process.
+
+## Convert and Quantize with Docker
+
+Run `docker run --rm -v /path/to/model/repo:/repo ollama/quantize -q quantlevel /repo`. For instance, if you have downloaded the latest Mistral 7B model, then clone it to your machine. Then change into that directory and you can run 
```

Comment:
The repo for quantize docker image is https://github.com/mxyng/quantize if you want to link it